moveit_setup_assistant/src/widgets/start_screen_widget.cpp

```diff
@@ -59,6 +59,11 @@
 // MoveIt
 #include <moveit/rdf_loader/rdf_loader.h>
 
+#ifdef _WIN32
+#include <direct.h>
+#define chdir _chdir
+#endif
+
```

Comment:
Using `chdir()` is just generating compile warning (as the example below), and it is not really a show-stopper, so I decided to remove the macro redefine to avoid confusion.

```
warning C4996: 'chdir': The POSIX name for this item is deprecated. Instead, use the ISO C and C++ conformant name: _chdir. See online help for details.
```